.travis.yml

```diff
@@ -63,6 +63,7 @@ script:
   - "python -m unittest hdf_compass.filesystem_model.test"
   - "python -m unittest hdf_compass.hdf5_model.test"
   - "python -m unittest hdf_compass.opendap_model.test"
+  - "python -m unittest hdf_compass.adios_model.test"
```

Comment:
I will add some install lines for it, otherwise I can still build a conda recipe